guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -68,39 +74,70 @@ If you lose connection to the command shell where the upgrade command is running
 ----
 # {foreman-maintain} upgrade run --target-version {TargetVersionMaintainUpgrade}
 ----
-
+endif::[]
+ifdef::katello[]
+. Check for running tasks
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake katello:upgrade_check
+----
+. Update operating system packages
++
+[options="nowrap" subs="attributes"]
+----
+# yum update -y
+----
+. Update repositories
++
+.For Centos 7 or Red Hat Enterprise Linux Users:
+[options="nowrap" subs="attributes"]
+----
+# yum update -y https://fedorapeople.org/groups/katello/releases/yum/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
```

Comment:
I know @evgeni is working on moving this. Perhaps https://yum.theforeman.org/katello/ should be used already?

Reply:
Yes please. 3.17, 3.18 and 4.0 is installable from yum.tfm today.